contrib/epee/include/serialization/wire/field.h

```diff
@@ -51,7 +51,11 @@ namespace wire
     `value_type` is `T` with optional `std::reference_wrapper` removed.
     `value_type` needs a `read_bytes` function when parsing with a
     `wire::reader` - see `read.h` for more info. `value_type` needs a
+<<<<<<< HEAD
```

Comment:
Botched rebase?

Reply:
Pushed an update to fix this merge issue.